docs/guide/exe_verilator.rst

```diff
@@ -129,6 +129,14 @@ Summary:
    is also used). Verilator manages the build itself, and for this --build
    requires GNU Make to be available on the platform.
 
+.. option:: --build-jobs [<value>]
+
+   Specify the level of parallelism for :vlopt:`--build`. The <value> must
+   be a positive integer specifying the maximum number of parallel build
+   jobs.
```

Comment:
```suggestion
.. option:: --build-jobs <value>

   Specify the level of parallelism for :vlopt:`--build`. If zero, uses the
   number of threads in the current hardware. Otherwise, the <value> must
   be a positive integer specifying the maximum number of parallel build
   jobs.
```

And fix -j similar.